spring-session/src/main/java/org/springframework/session/MapSession.java

```diff
@@ -40,7 +40,7 @@
  * @since 1.0
  * @author Rob Winch
  */
-public final class MapSession implements ExpiringSession, Serializable {
+public class MapSession implements ExpiringSession, Serializable {
```

Comment:
First it appeared unnecessary for the changes that were made. Does anything need this change right now? Second by keeping it final we are able to change certain implementation details without worrying about passivity.